### ISSUE TITLE
[$20][MSFT-184] Id attribute value must be unique

### ISSUE THREAD

**Issue by veshu** [topcoder-platform/community-app#2852]
**Steps to Reproduce**
1. Navigate to https://www.topcoder.com/challenges/30095347
2. Right click > Inspect > Elements
3. Check 
<link href="https://d2nl5eqipnb33q.cloudfront.net/static-assets/main-1562213638466.css" id="tru-style" rel="stylesheet">
<link data-chunk="challenge-details/chunk" id="tru-style" href="https://d2nl5eqipnb33q.cloudfront.net/static-assets/challenge-details/chunk-1562213638466.css" rel="stylesheet">
<link data-chunk="topcoder-website/chunk" id="tru-style" href="https://d2nl5eqipnb33q.cloudfront.net/static-assets/topcoder-website/chunk-1562213638466.css" rel="stylesheet">

**Expected Result**
Ids should be unique

**Actual Result**
All three elements have the same id

**Device**: Desktop/Labtop
**Operating System**: macOS
**Browser**: Chrome

**WCAG Category**: 4. Robust
**WCAG Level**: A
**WCAG Success Criterion**: 4.1.1 Parsing
**Screen Reader Used**: N/A
**Accessibility Audit Tool used**: Other

**Attachment**
https://topcodermsft-my.sharepoint.com/personal/pd-topcoder_topcodermsft_onmicrosoft_com/Documents/Forms/All.aspx?cid=37e62274%2D78ab%2D496a%2D918f%2Dabba8f38f3fe&FolderCTID=0x0120005C598A51049FC14CBA882E1AEE168F51&id=%2Fpersonal%2Fpd%2Dtopcoder%5Ftopcodermsft%5Fonmicrosoft%5Fcom%2FDocuments%2FMSFT%2DTeams%2DQA%2FJune%202019%2F29%20Jun%2FTest%20Results%2FBug%20Videos%20%26%20Log%20Files%2Fyoctocode%2Fbug%206

**Comment by ThomasKranitsas** [comment]
Contest https://www.topcoder.com/challenges/30095990 has been created for this ticket.<br/><br/>```This is an automated message for thomaskranitsas via Topcoder X```

**Comment by ThomasKranitsas** [comment]
Contest https://www.topcoder.com/challenges/30095990 has been updated - the new changes has been updated for this ticket.<br/><br/>```This is an automated message for thomaskranitsas via Topcoder X```

**Comment by asadath1395** [comment]
@mishacucicea This needs change in topcoder-react-utils, is it in scope?

**Comment by veshu** [comment]
@asadath1395 Yes please.

**Comment by cwdcwd** [comment]
Contest https://www.topcoder.com/challenges/30095990 has been updated - the new changes has been updated for this ticket.<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by cwdcwd** [comment]
Contest https://www.topcoder.com/challenges/30095990 has been updated - it has been assigned to anonymousjaggu.<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by anonymousjaggu** [comment]
@veshu shall I make merge request in the `topcoder-react-utils` project for this?

**Comment by cwdcwd** [comment]
Contest https://www.topcoder.com/challenges/30095990 has been updated - it has been assigned to anonymousjaggu.<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by anonymousjaggu** [comment]
PR sent here https://github.com/topcoder-platform/topcoder-react-utils/pull/41

**Comment by cwdcwd** [comment]
Contest https://www.topcoder.com/challenges/30097072 has been created for this ticket.<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by cwdcwd** [comment]
Contest https://www.topcoder.com/challenges/30097072 has been updated - it has been assigned to anonymousjaggu.<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by sexta13** [comment]
this issue is related with this: 
https://github.com/topcoder-platform/community-app/issues/3095
The PR https://github.com/topcoder-platform/topcoder-react-utils/pull/43 solves both

**Comment by veshu** [comment]
@drasticdpk Could you please confirm this and close.

**Comment by tosha5252** [comment]
If I understand correctly, this will be changed when the other issue is resolved? So this one should be closed? 

![tru-style](https://user-images.githubusercontent.com/6400904/62531528-434ed600-b808-11e9-9f9a-c8da2e7a0a33.png)

**Comment by crazyk07** [comment]
Closing ticket due to shift in priority, will not be able to complete but will still pay for effort rendered.

Note: Ticket was dependent on react Utils

**Comment by crazyk07** [comment]
Payment task has been updated: https://software.topcoder.com/review/actions/ViewProjectDetails?pid=30097072<br/><br/>```This is an automated message for crazyk via Topcoder X```
